source/adminguide/templates.rst

```diff
@@ -249,7 +237,12 @@ To upload a template:
       available, or All Zones to make it available throughout
       CloudStack.
 
-   -  **OS Type**: (Except for VMware). This helps CloudStack and the hypervisor perform
+   - **Read VM settings from OVA**. (VMware only) If selected, the registered template will allow users to deploy VMs as clones of the template, including all their properties, configurations, end-user license agreements, disks, os type, etc. This option allow users to register virtual appliances. See `Support for Virtual Appliances <virtual_machines.html#about-virtual-appliances>`_.
```

Comment:
This option **allow** :arrow_right:  **allows**

Reply:
Thanks @GabrielBrascher - fixed